tests/subsys/logging/dictionary/pytest/test_logging_dictionary.py

```diff
@@ -38,8 +38,10 @@ def process_logs(dut: DeviceAdapter, build_dir):
     logger.info(f'Dictionary JSON: {dictionary_json}')
 
     # Read the encoded logs and save them to a file
-    # as the log parser requires file as input
-    handler_output = dut.readlines_until(regex = '^##ZLOGV1##[0-9]+', timeout = 10.0)
+    # as the log parser requires file as input.
+    # Timeout is intentionally long. Twister will
+    # timeout earlier with per-test timeout.
+    handler_output = dut.readlines_until(regex = '.*##ZLOGV1##[0-9]+', timeout = 600.0)
```

Comment:
is this timeout too long (this test could executes ca. 10 min after changes)?

Reply:
Twister has its own timeout outside of this which is shorter, and will terminate the test accordingly. So a large number here doesn't change that aspect of twister.